packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -500,7 +500,6 @@ const disableVmHighAvailability = async (xapi: Xapi, vm: Vm) => {
 //
 // job.start(data: { mode: Mode, reportWhen: ReportWhen })
 // ├─ task.info(message: 'vms', data: { vms: string[] })
-// ├─ task.warning(message: 'missingVms', data: { vms: string[] })
```

Comment:
What should be done about the handling of this in xo-web?

Reply:
There nothing to be done in the `xo-web` part, tasks warning are handled generically and can exists on all deep levels.